.gitignore

```diff
@@ -0,0 +1,27 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/oje/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/oje/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local

```

Comment:
if some additional developer will work with you how should he know the variables which you add on your local machine?